### PR TITLE
Add build script

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/WATT/issues/216
[Problem] Lack of build script for tau 1.0 which WATT requires
	to build TAU
[Solution] add build script

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Tizen Advanced UI",
   "version": "1.0.25",
   "scripts": {
+    "build": "npx grunt build",
```
Comment (lslachciak) [inline package.json]: grunt build
will not be enough?
in 1.1 it is done this way
https://github.com/Samsung/TAU/blob/tau_1.1/package.json

Comment (grzegorz-czajkowski) [reply package.json]: Ok I will check this.

Comment (grzegorz-czajkowski) [reply package.json]: You're right. I also updated README.md accordingly.

**Review by lslachciak — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -35,19 +35,19 @@ nvm use 6.16.0
 
 Build TAU by running the following command in the tau directory:
 
-`npx grunt build`
```
Comment (lslachciak) [inline README.md:38]: npm run build

**Review by lslachciak — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -35,19 +35,19 @@ nvm use 6.16.0
...
+`grunt build`
 
 ## Running the tests
 
 TAU has automated tests. To run tests execute command:
 
-`npx grunt test`
+`grunt test`
```
Comment (lslachciak) [inline README.md]: npm run test

File: README.md [hunk 4]
```diff
@@ -35,19 +35,19 @@ nvm use 6.16.0
...
 
 ## Debuggging
 
 In order to set additional logging in TAU you need to build TAU with --tau-debug option
 
-`npx grunt build --tau-debug`
+`grunt build --tau-debug`
```
Comment (lslachciak) [inline README.md]: npx grunt build --tau-debug
I think should stay.
AFAIK I can not pass argument to npm command so 
npm run build --tau-debug won't work

**Review by lslachciak — CHANGES_REQUESTED** [review]
please check comments for README see also
https://github.com/Samsung/TAU/blob/tau_1.1/README.md

File: README.md (hunk 2 above)
Comment (grzegorz-czajkowski) [reply README.md:38]: Done.

File: README.md (hunk 3 above)
Comment (grzegorz-czajkowski) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (grzegorz-czajkowski) [reply README.md]: Done.

**Review by lslachciak — APPROVED** [review]

**Pull request merged**